packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -113,6 +113,11 @@ const propTypes = {
    * The value must be in the `YYYY-MM-DD` format or the all-numeric date format based on the locale.
    */
   value: PropTypes.string,
+  /**
+   * @private
+   * Prop to show inline version of date picker component.
+   */
+  inline: PropTypes.bool,
```

Comment:
Should this be named like a boolean to align with our naming standards? We can always map to inline
```suggestion
  isInline: PropTypes.bool,
```

Reply:
updated : [63be00c147](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-06f07d3cca7012107347dc63be00c147)